backend/src/main/java/com/redhat/cloud/notifications/db/repositories/BehaviorGroupRepository.java

```diff
@@ -186,7 +186,7 @@ public boolean updateEventTypeBehaviors(String accountId, UUID eventTypeId, Set<
         // First, let's make sure the event type exists.
         EventType eventType = entityManager.find(EventType.class, eventTypeId);
         if (eventType == null) {
-            return false;
+            throw new NotFoundException("Event type not found");
```

Comment:
This does not change the application behavior as we already returned a 404 if the event type was not found. However, it's still a good change because we return a better error message and it uses the same pattern we use in several other places. As a consequence, it's normal if this doesn't break any existing test.